site/en/docs/extensions/mv3/intro/mv3-overview/index.md

```diff
@@ -145,16 +128,16 @@ There are a number of other changes introduced in Manifest V3:
 * [Web accessible resources](/docs/extensions/mv3/intro/mv3-migration#web-accessible-resources): These resources are now available only to specified sites and extensions.
 * [Content security policy (CSP)](/docs/extensions/mv3/intro/mv3-migration#content-security-policy): You now specify separate CSP for different execution contexts in a single object, and certain policies are disallowed.
 * [executeScript() changes](/docs/extensions/mv3/intro/mv3-migration#executing-arbitrary-strings): Extensions can no longer execute arbitrary strings, only script files and functions. This method is also migrating from the Tabs API to the new Scripting API.
-* [A new in-memory StorageArea][2] that can be used to store values in-memory across service worker restarts.
-
-The following features will be added to Manifest V3 soon:
-
-* **Dynamic content scripts:** the new [Scripting API][1] lets extensions register and unregister content scripts at runtime.
-* **New favicon API:** this new JavaScript API replaces "chrome://favicons" and gives  developers a way to retrieve websites' favicons.
-
-Look for announcements of these and other Manifest V3 features as they become available.
-
-[1]: /docs/extensions/reference/scripting/
-[2]: /docs/extensions/reference/storage/#property-session
-[3]: https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise
-[4]: /docs/extensions/mv3/promises#compare-to-callback
+* [A new in-memory StorageArea][storage-session] that can be used to store values in-memory across service worker restarts.
```

Comment:
Please make items in the list match each other grammatically.
"Express coordinate ideas in similar form." (Strunk and White, rule II-19.)

Reply:
👍